src/main.css

```
@@ -23,4 +23,12 @@
       0 20%
     );
   }
+
+  .perspective {
+    perspective: 350px;
+  }
+
+  .animation-delay-4000 {
+    animation-delay: 40s;
+  }
```

```suggestion
  .animation-delay-40000 {
    animation-delay: 40s;
  }
```
Soit `-40` en secondes, soit `-40000` en ms